<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="labelPesanan">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mulish</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pesanan</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="paket_box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>631</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29, 45, 94); border-radius: 10px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="paketText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Paket</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="labelbayar">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>250</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29, 45, 94); border-radius: 10px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="bayar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Pembayaran</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="tagihan">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>310</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(232, 232, 232);</string>
   </property>
   <widget class="QLabel" name="harga">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rp  XXX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tagihantitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tagihan</string>
    </property>
   </widget>
   <widget class="QLabel" name="rincian">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Paket XCV - Y Hari</string>
    </property>
   </widget>
   <widget class="QLabel" name="paymentsuccess">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 204, 57);</string>
    </property>
    <property name="text">
     <string>Pembayaran Berhasil</string>
    </property>
   </widget>
   <widget class="QLabel" name="bank">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>BCA Virtual Account</string>
    </property>
   </widget>
   <widget class="QLabel" name="rekening">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>9876543210987654</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Booking Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="bookingNumber">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>xx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="framePaket">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>631</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="jmlCV">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X CV</string>
    </property>
   </widget>
   <widget class="QLabel" name="jmlHari">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>X Hari</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 12pt &quot;Mulish ExtraBold&quot;;
color: white;
background-color: rgb(29, 45, 94);
border-radius: 10px
</string>
    </property>
    <property name="text">
     <string>Unggah CV</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileName">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Mulish&quot;;</string>
    </property>
    <property name="text">
     <string>No File Uploaded!</string>
    </property>
   </widget>
   <widget class="QLabel" name="prosesReview">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Mulish&quot;; color:rgb(255, 153, 0)</string>
    </property>
    <property name="text">
     <string>Proses Review</string>
    </property>
   </widget>
   <widget class="QLabel" name="gambar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../img/resume (1).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="submitBookingButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>700</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 12pt &quot;Mulish ExtraBold&quot;;
color: white;
background-color: rgb(29, 45, 94);
border-radius: 10px
</string>
   </property>
   <property name="text">
    <string>Submit Booking</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 12pt &quot;Mulish ExtraBold&quot;;
color: white;
border-radius: 10px;
background-color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homescreen">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>700</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 12pt &quot;Mulish ExtraBold&quot;;
color: white;
background-color: rgb(29, 45, 94);
border-radius: 10px
</string>
   </property>
   <property name="text">
    <string>Back To Home Screen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="layananKami">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 12pt &quot;Mulish ExtraBold&quot;;
background-color: rgb(255, 255, 255);
border-radius: 20px</string>
   </property>
   <property name="text">
    <string>Layanan Kami</string>
   </property>
  </widget>
  <widget class="QPushButton" name="layananKami_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 12pt &quot;Mulish ExtraBold&quot;;
background-color: rgb(255, 255, 255);
border-radius: 20px</string>
   </property>
   <property name="text">
    <string>Tentang Kami</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1601</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../img/logo aksel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>labelPesanan</zorder>
  <zorder>paket_box</zorder>
  <zorder>labelbayar</zorder>
  <zorder>tagihan</zorder>
  <zorder>framePaket</zorder>
  <zorder>submitBookingButton</zorder>
  <zorder>delete_button</zorder>
  <zorder>homescreen</zorder>
  <zorder>layananKami</zorder>
  <zorder>layananKami_2</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources>
  <include location="../../img/cv.qrc"/>
 </resources>
 <connections/>
</ui>
